docker/safe/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
+
+# create (safe)user with a home directory
+ENV UID 1000
+ENV NB_USER polly
+
+RUN adduser --disabled-password \
+    --gecos "Default user" \
+    --uid ${UID} \
+    ${NB_USER}
+
+# allow user access to the WORKDIR
+RUN chown -R ${USER}:${USER} /opt/
```

Comment:
can be a bit more restrictive by only changing `/opt/polynote` or `/opt/polynote/notebooks/`... thoughts?

Reply:
What are the downsides to allowing the user to use `/opt/`? What would extra restrictions buy us here, as the permissions are set on the directory inside the container - or am I missing something?